doc/generic/pgf/text-en/pgfmanual-en-tikz-graphs.tex

```diff
@@ -243,8 +243,9 @@ \subsubsection{Concept: Chain Groups}
 
 As can be seen, the placement is not particularly nice by default, use the
 algorithms from the graph drawing libraries to get a better layout. For
-instance, adding |tree layout| to the above code results in the following
-somewhat more pleasing rendering:
+instance, adding |tree layout| to the above code (and
```

Comment:
didn't you fix this already in #881 ?

Reply:
@ilayn, yes, I did. Have a look at https://github.com/pgf-tikz/pgf/pull/883#discussion_r446149566 for the reason of that.